tests/js/server/aql/aql-persistent-index.js

```diff
@@ -30,20 +30,64 @@ const internal = require("internal");
 const jsunity = require("jsunity");
 const helper = require("@arangodb/aql-helper");
 const getQueryResults = helper.getQueryResults;
-  
+const isCluster = require('internal').isCluster();
+
 const cn = "UnitTestsCollection";
-  
+const idxName = "testIdx";
+const query1 = `FOR d IN ${cn} FILTER d.value == 1 SORT d._key RETURN d`;
+const query2 = `FOR d IN ${cn} FILTER d.value == 2 SORT d._key RETURN d`;
+
+
 let explain = function (query, params) {
   return helper.removeClusterNodes(helper.getCompactPlan(AQL_EXPLAIN(query, params, { optimizer: { rules: [ "-all", "+use-indexes" ] } })).map(function(node) { return node.type; }));
 };
 
+let testandValidate = function (query, isUnique, insertMoreDocs, insertIndexBeforeDocs, sameValue, isPrimary, selectivity) {
+  let ensureIndex = () => {
+    internal.db[cn].ensureIndex({type: "persistent", name: idxName, unique: isUnique, fields: ["value"]});
+  };
+
+  if (insertMoreDocs) {
+    if (insertIndexBeforeDocs) {
```

Comment:
Do we actually need to test the before/after index creation in our tests here?
I don't see the benefit of testing it here, because we will be waiting for all estimates to be synced anyway.
To simplify, we could just create the index in a way that we don't need to wait for the estimates to settle.